### PR TITLE
FS - CircleCI Setup

### PR DESCRIPTION
**Pull request by agnieszkajacek**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      - rspec:
+          matrix:
+            parameters:
+              database:
+                - mysql
+                - postgres
+              gemfile:
+                - spree_fullscript
+              ruby-version:
+                - "2.5.3"
```
Comment (GrzegorzStochel) [inline .circleci/config.yml:81]: we need ruby 2.7.3

Comment (agnieszkajacek) [reply .circleci/config.yml:82]: Added

**Comment by Vegann** [comment]
LGTM

**Comment by helsayed** [comment]
LGTM

**Comment by agnieszkajacek** [comment]
@damianlegawiec Could you merge it, please?

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**